src/app/features/genes/components/gene-soe-charts/gene-soe-charts.component.ts

```diff
@@ -73,6 +77,13 @@ export class GeneSoeChartsComponent implements OnInit {
     });
   }
 
+  getBarColor(title: string) {
+    if (title === 'Genetic Risk Score') {
```

Comment:
The color is determined by whether the score chart is the "Genetic Risk Score".  Green in this case but otherwise lavender.

Reply:
The Target Risk Score should be the one with the different color, not the Genetic Risk Score

Reply:
Fixed!